modules/reference/pages/data-transform-api.adoc

```diff
@@ -95,7 +93,274 @@ type WriteEvent interface {
 
 The `WriteEvent` type contains information about a record that was written.
 
+==== RecordWriter
+
+```go
+type RecordWriter interface {
+      // Write writes a record to the output topic.
```

Comment:
```suggestion
      // Writes a record to the output topic.
```

Reply:
This is a standard golang pattern, as awkward as it is: https://go.dev/blog/godoc

Quote:

> Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.